common.py

```diff
@@ -1062,7 +1089,9 @@ def calc_bdt_scores(X, model_file=bdt_model_file):
         score = xgb_model.predict_proba(X)[:,1]
     return score
 
-def apply_bdtbased(cols,wp,lumi,anti=False):
+def apply_bdtbased(cols,wp,lumi,anti=False,xrootd_url='root://cmseos.fnal.gov//store/user/lpcdarkqcd/boosted/BDT_based/'):
+
+    check_if_model_exists(ddt_map_file, xrootd_url)
```

Comment:
1. also apply this to `bdt_model_file`
2. make both `bdt_model_file` and `ddt_map_file` configurable arguments of this function rather than hardcoded

Reply:
I removed the ddt_map as a global variable and have it configurable. I left bdt_model_file as a global variable, but additionally have it configurable at a function level.

Reply:
`check_if_model_exists()` should still be applied to `bdt_model_file`.

Reply:
Done